### PR TITLE
TCES-34: Implement a button to add more entries to the timeline locally

### PR DESCRIPTION
**Pull request by thardchi**
Link to issue: https://github.com/uoftblueprint/tces/issues/34
 
# Description

Implemented a functional component called ActivityTimline in src/ActivityTimeline.js I added a text field and a button so users can add more entries to the timeline.

Fixes # 34

## Sample curl
Ignore if PR does not affect an endpoint.
## Response
Ignore if PR does not affect an endpoint.

## Type of change

Please delete options that are not relevant.

- [x] New feature (non-breaking change which adds functionality)

# Checklist:

- [ ] I have run `npm run lint:fix`
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation (if required)
- [x] My changes generate no new warnings

Screen Shot 2022-12-04 at 9 36 23 AM
Screen Shot 2022-12-04 at 9 36 41 AM

### CODE REVIEW

**Review by jordanjanakievski — CHANGES_REQUESTED** [review]
Everything is working as demonstrated and expected.

I spoke with Edward and if you can import the feature into App.tsx similar to how Communication was imported and displayed. This will make it easy for people to the code run and find your feature immediately.

File: client/src/components/ActivityTimeline.js [hunk 1]
```diff
@@ -0,0 +1,58 @@
+import React, {useState} from 'react'
```
Comment (ganeshasapu) [inline client/src/components/ActivityTimeline.js:1]: I think this file should be a ts file instead of js

**Review by ganeshasapu — CHANGES_REQUESTED** [review]
Super cool component! Everything works as intended, just one small fix. Also I can't believe I've never used Math.random() to generate ids--it seems like a really good idea.

**Review by jordanjanakievski — APPROVED** [review]
Update to App.tsx looks good.

**Review by ganeshasapu — APPROVED** [review]
Everything looks good!

**Pull request merged**